setup.cfg

```diff
@@ -1,2 +1,7 @@
 [flake8]
 ignore=E501
+
+[pytest]
+addopts=-r=fEsxXw
+base_url=https://www-dev.allizom.org
```

Comment:
As mentioned in IRC, this doesn't seem to be working for me. If I don't specify a --base-url on the command line I end up with base-url == `None`